src/aibtc-dao/extensions/treasury/public/deposit-ft.ts

```diff
@@ -77,6 +79,14 @@ async function main() {
     CONFIG.ACCOUNT_INDEX
   );
   const nextPossibleNonce = await getNextNonce(CONFIG.NETWORK, address);
+
+  // ### POST CONDITIONS ADDED ###
+  const postConditions = [
+    Pc.principal(address)
+      .willSendEq(args.amount)
+      .ft(args.ftContract as `${string}.${string}`, "token")
```

Comment:
Do we need to force this type assertion?

If we have to match that type, maybe use the deconstructed values?
```ts
      .ft(`${ftAddress}.${ftName}`, "token")
```